io.gitlab.android_translation_layer.BaseApp.yml

```diff
@@ -0,0 +1,261 @@
+id: io.gitlab.android_translation_layer.BaseApp
+runtime: org.gnome.Platform
```

Comment:
is GNOME really necessary?

Reply:
Yes, we use GTK4. The android View hierarchy will be translated into a GTK4 widget hierarchy. Also we use WebkitGTK for Android apps requiring webkit